session_4/blogspot/app/views/articles/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<h1>Articles</h1>
+    <div class="row">
+      <% @articles.each do |article| %>
+        <%if @im || article.public || article.user_id == @id%>
```

Comment:
You can show the private articles in the index view but not the contents of a private article on show view

Reply:
Ok, I'll change them.